appraisal.lic

```diff
@@ -25,6 +26,42 @@ class Appraisal
     @equipment_manager.items.each { |item| break unless appraise_item?(item) }
     true
   end
+  
+  def train_appraisal_with_pouches?
+    settings = get_settings
+    return false unless settings.train_appraisal_with_pouches
+    full_pouch_container = settings.full_pouch_container
+    if !full_pouch_container
+      echo "Must set full_pouch_container in YAML"
+      return false
+    end      
+    if full_pouch_container == settings.spare_gem_pouch_container
+      echo "*** Cannot have full pouches be stored where spare pouches are ***"
+      return false
+    end
+    
+    pouches = rummage('C pouch', full_pouch_container)
+    pouches.each_with_index do |p, i|
```

Comment:
Since you're ending your iteration on the size of ordinals, and only use your index to look up ordinals, have you considered just looping over ordinals and trying to get/app each ordinal, and just breaking on mind lock or no pouch found for that ordinal?